devtools/conda-recipe/meta.yaml

```diff
@@ -33,12 +33,13 @@ test:
     - ipyext
 
   source_files:
-    - mbuild/examples/*
-
+    - .coveragerc
+  
   commands:
-    - py.test -v --pyargs mbuild
-    - cd mbuild/examples
-    - py.test test_examples.py -v
+    - conda install -yc conda-forge python-coveralls 
+    - conda install -yc conda-forge pytest-cov
```

Comment:
Ok there's no windows py34 package on conda-forge. I'm just going to build our own version and put it in the mosdef-hub channel for now.